azimuth_capi/config.py

```diff
@@ -207,6 +207,9 @@ class Configuration(
     #: The number of seconds to wait between timer executions
     timer_interval: conint(gt = 0) = 60
 
+    #: The number of minutes to wait befoore marking a cluster as unhealthy
+    cluster_timeout_minutes: conint(gt = 0) = 30
```

Comment:
Can we make this seconds like all other places where intervals are configured?

Reply:
yes... I suppose, for consistency that makes sense.